### DISCUSSION TITLE
Nebula Book and Documentation

### DISCUSSION THREAD

**Discussion by wey-gu** [vesoft-inc/nebula#4318] [Announcements]
Do you know Nebula Graph Documentation is also a book distributed in PDF?

**How to get it?**
- Checkout the Nebula Graph documentation here 👉🏻 https://docs.nebula-graph.io/
- Scroll down to the bottom of the page, you will then find the PDF URL of the book.

<img width="1344" alt="nebula_book" src="https://user-images.githubusercontent.com/1651790/173331987-0fd4af7d-bdf2-4f74-a0ad-a54450682005.png">

> The book is versioned, too. i.e. its 3.1.0 version is 👉🏻 [docs.nebula-graph.io/3.1.0/pdf/NebulaGraph-EN.pdf](https://docs.nebula-graph.io/3.1.0/pdf/NebulaGraph-EN.pdf)

**Comment by fly-chosen** [comment]
get
